tests/ci_install.sh

```diff
@@ -1,9 +1,31 @@
 set -x   # Show which command is being run
 
-echo "Travis build stage: $TRAVIS_BUILD_STAGE_NAME"
+case ${RUNNER_OS} in
+linux|Linux)
+    sudo apt-get install \
+      libhdf5-serial-dev \
+      libnetcdf-dev \
+      libproj-dev \
+      proj-data \
+      proj-bin \
+      libgeos-dev \
+      libopenmpi-dev
+    ;;
+osx|macOS)
+    sudo mkdir -p /usr/local/man
+    sudo chown -R "${USER}:admin" /usr/local/man
+    HOMEBREW_NO_AUTO_UPDATE=1 brew install hdf5 proj geos open-mpi netcdf ccache
+    ;;
+esac
+
+mkdir -p $HOME/.config/yt
+echo "[yt]" > $HOME/.config/yt/ytrc
+echo "suppressStreamLogging = True" >> $HOME/.config/yt/ytrc
+cat $HOME/.config/yt/ytrc
+cp tests/matplotlibrc .
```

Comment:
Is this strictly part of the migration process ? I can't find where this came from or why it's a necessary step.

Reply:
It's not really part of migration process, it's always been there. It's our way to setup a proper matplotlib backend to something that's non-interactive / doesn't open windows.

Reply:
ok ! I do think a comment would be nice though :)